assets-and-images.md

```diff
@@ -53,17 +57,30 @@ different locales or regions, reading directions, etc.
 
 ## Working with assets
 
-Your application can access its assets through the
-[AssetBundle](http://docs.flutter.io/flutter/services/AssetBundle-class.html)
-interface. The two main methods on an asset bundle allow you to load a string
+Your application can access its assets through an
+[AssetBundle](http://docs.flutter.io/flutter/services/AssetBundle-class.html).
+The two main methods on an asset bundle allow you to load a string
 (`loadString`) or an image (`loadImage`) out of the bundle, given a logical key.
```

Comment:
Does the extracted string represent the content of the data/config file? If so, please say this explicitly. What happens when the loadString() method is called on a image or icon key, or if loadImage() is called on a string-only data file?